### PR TITLE
Hw28

### PR DESCRIPTION
**Pull request by andrjuxa201185**
@iliatcymbal  PTAL

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -45,7 +45,9 @@
     "react/jsx-one-expression-per-line": 0,
     "react/button-has-type": 0,
     "jsx-a11y/click-events-have-key-events": 0,
-    "require-yield": 0
+    "require-yield": 0,
```
Comment (iliatcymbal) [inline .eslintrc.json]: what's for this rule?

File: .idea/andrey_podgorniy_shop.iml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (iliatcymbal) [inline .idea/andrey_podgorniy_shop.iml]: no need this file

File: .idea/misc.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (iliatcymbal) [inline .idea/misc.xml]: no need this file

File: .idea/modules.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (iliatcymbal) [inline .idea/modules.xml]: no need this file

File: .idea/vcs.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (iliatcymbal) [inline .idea/vcs.xml]: no need this file

File: .idea/workspace.xml [hunk 6]
```diff
@@ -0,0 +1,269 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (iliatcymbal) [inline .idea/workspace.xml]: no need this file

File: src/pages/products/Products.js [hunk 7]
```diff
@@ -1,18 +1,14 @@
...
-  getProductsService,
-  deleteProductService,
-  updateProductsService
-} from '../../services/productService';
 import './products.scss';
-import { setProducts } from '../../store/products';
-import { FaEdit, FaRegTrashAlt } from 'react-icons/fa';
+import {
+  updateProductAsync,
+  setProductsAsunc,
```
Comment (iliatcymbal) [inline src/pages/products/Products.js]: asYnc

**Comment by andrjuxa201185** [comment]
@iliatcymbal PTAL

File: src/store/user/sagas.js [hunk 8]
```diff
@@ -20,11 +21,12 @@ function* fetchUser() {
 }
```
Comment (iliatcymbal) [inline src/store/user/sagas.js]: what about an error here?

**Pull request merged**